Fw/Types/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ set(UT_SOURCE_FILES
   "${CMAKE_CURRENT_LIST_DIR}/test/ut/TypesTest.cpp"
 )
 set(UT_MOD_DEPS
-  "${FPRIME_FRAMEWORK_PATH}/Os"
+  Os
```

Comment:
Looks like the proposed change would be a breaking change for projects that may have been defining `MOD_DEPS` using full paths like this.